WalletWasabi/BitcoinCore/P2pNode.cs

```diff
@@ -86,6 +88,31 @@ public async Task ConnectAsync(CancellationToken cancel)
 			}
 		}
 
+		public IEnumerable<Transaction> GetMempoolTransactions(IEnumerable<uint256> txids, CancellationToken cancel)
+		{
+			var node = Guard.NotNull(nameof(Node), Node);
+
+			try
+			{
+				TrustedP2pBehavior.DisableProcess(txids);
+
+				// Chunk it because it doesn't work properly for more.
```

Comment:
I think it is because of a DoS-related limitation. Interesting I did a search and there is no information regarding that so probably I was wrong. It should work. Disregard this comment I just shared my findings. 

https://github.com/MetacoSA/NBitcoin/blob/master/NBitcoin/Protocol/Node.cs#L1625